cmd/homi/extra/cmd.go

```diff
@@ -21,23 +21,22 @@ import (
 	"fmt"
 	"os"
 
-	"gopkg.in/urfave/cli.v1"
-
 	"github.com/klaytn/klaytn/common"
 	"github.com/naoina/toml"
+	"github.com/urfave/cli/v2"
 )
 
 var ExtraCommand = cli.Command{
 	Name:  "extra",
```

Comment:
How about declaring it as `*cli.Command` type rather than adding `&`  for all usage? Not only for ExtraCommand, the command can be applied to all Command declarations.

Reply:
Also for Flag declarations